test/UnitTest/Microsoft.AspNetCore.OData.Test/PublicApi/Microsoft.AspNetCore.OData.PublicApi.bsl

```diff
@@ -3429,7 +3481,19 @@ public class Microsoft.AspNet.OData.Formatter.Deserialization.ODataResourceSetDe
 	public virtual System.Threading.Tasks.Task`1[[System.Object]] ReadAsync (Microsoft.OData.ODataMessageReader messageReader, System.Type type, ODataDeserializerContext readContext)
 
 	public virtual object ReadInline (object item, Microsoft.OData.Edm.IEdmTypeReference edmType, ODataDeserializerContext readContext)
-	public virtual System.Collections.IEnumerable ReadResourceSet (ODataResourceSetWrapper resourceSet, Microsoft.OData.Edm.IEdmStructuredTypeReference elementType, ODataDeserializerContext readContext)
```

Comment:
>ODataResourceSetWrapper [](start = 64, length = 23)

Technically, I think this is a breaking change -- if someone has overridden this virtual method in a derived class then they will be expecting an ODataResourceSetWrapper, not an ODataResourceSetWrapperBase. We should think through scenarios in which users overwrite the ODataResourceSetDeserializer and whether this is likely to break them.